.github/workflows/build_and_test.yml

```diff
@@ -99,10 +99,14 @@ jobs:
       run: git merge --progress --ff-only origin/${{ github.event.inputs.target }}
     # Cache local repositories. Note that GitHub Actions cache has a 2G limit.
     - name: Cache Scala, SBT, Maven and Zinc
-      uses: actions/cache@v1
+      uses: actions/cache@v2
```

Comment:
why is the difference between v1 and v2?

Reply:
In order to allow multiple file path (https://github.com/actions/cache#whats-new). It is also to make it consistent in this file. `cache@v2` is used in all places at this file except here.

Reply:
Previously, other files like `build/mvn` and `build/sbt` are also cached and overwritten. So, when you have some changes there, they are ignored.